evals/test_runners/utils.py

```diff
@@ -30,3 +31,12 @@ def exec_code_and_get_globals_and_output(code: str) -> Tuple[Dict[str, Any], str
         raise e
     
     return globals, actual_output_str
+
+
+def remove_process_pausing_code_lines(code: str) -> str:
```

Comment:
Love this. Feels so unnatural to make a plot and not do `plt.show()`.

Reply:
Yeah. I created this because the AI code was sometimes returning `plt.show()` in it which was causing the tests to stop running.